target/linux/ath79/dts/ar9344_wd_mynet-n750.dts

```diff
@@ -0,0 +1,216 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar9344.dtsi"
+
+/ {
+	model = "Western Digital My Net N750";
+	compatible = "wd,mynet-n750", "qca,ar9344";
+
+	chosen {
+		bootargs = "console=ttyS0,115200n8";
+	};
+
+	aliases {
+		led-boot = &power;
+		led-failsafe = &power;
+		led-running = &power;
+		led-upgrade = &power;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		wifi {
+			label = "wd:blue:wifi";
```

Comment:
LED naming scheme is {boardname}:{color}:{name} so it should be prefixed with mynet-n750 here.
TP-Link is an exception here because they have many boards sharing the same LED configuration.